### ISSUE TITLE
 Overwriting a file when reuploading does not seem to work

### ISSUE THREAD

**Issue by vincentducorps** [adonisjs/adonis-bodyparser#14]
## Package version
> "version": "4.1.0",
> "adonis-version": "4.1.0",

> @adonisjs/ace@5.0.8
> @adonisjs/ally@2.1.1
> @adonisjs/auth@3.0.7
> @adonisjs/bodyparser@2.0.5
> @adonisjs/cors@1.0.7
> @adonisjs/fold@4.0.9
> @adonisjs/framework@5.0.9
> @adonisjs/ignitor@2.0.8
> @adonisjs/lucid@6.1.3
> @adonisjs/session@1.0.27
> @adonisjs/shield@1.0.8

## Node.js and npm version
> NodeJS v10.9.0
> npm 6.2.0

## Sample Code (to reproduce the issue)
From [documentation](https://adonisjs.com/docs/4.1/file-uploads) :

> view 

```html
<form method="POST" action="upload" enctype="multipart/form-data">
  {{ csrfField() }}
  <input type="file" name="profile_pic" />
  <button type="submit"> Submit </button>
</form>
```

> start/route

```javascript
Route.post('upload', async ({ request }) => {
  const profilePic = request.file('profile_pic', {
    types: ['image'],
    size: '2mb'
  })

  await profilePic.move(Helpers.tmpPath('uploads'), {
    name: 'custom-name.jpg',
    overwrite: true
  })

  if (!profilePic.moved()) {
    return profilePic.error()
  }
  return 'File moved'
})
```

This 1st time you use this script, everything is going well.

When you try to upload another image to replace it, then there is an HTTP error 500: "dest already exists."

**Comment by tlgreg** [comment]
Looks like the docs has been updated before the package.
Its been added in 41b661ffee00b36c5b9ac6fa9a31e94d03a4e4aa which has not been released yet.

**Comment by thetutlage** [comment]
Released as `2.0.6`
